_includes/tutorials/kafka-connect-datagen-local/kafka/code/docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+---
+version: '2'
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:5.5.0
```

Comment:
We're using cp-kafka-connect-base:5.5.1 for the connector but 5.5.0 for the services.  Is that intentional?

Reply:
I'll change this to 5.5.0 for consistency.